src/actions/index.js

```diff
@@ -5,6 +7,13 @@ export const setCredit = (charges) => {
   };
 };
 
+export const addBatchCreditCharge = (charges) => {
+  return {
+    type: 'ADD_BATCH_CREDIT_CHARGE',
+    charges,
```

Comment:
Even not using https://github.com/acdlite/redux-actions I like the organization of having a standard for actions, but that's just me :P